documentation/pipeline.man

```diff
@@ -177,6 +177,10 @@ Argument is optional. Print the determined ra, de, and roll to \fIpath\fP. Defau
 \fB--compare-attitude\fP [\fIpath\fP]
 Argument is optional. Compare expected to actual attitude. Prints to \fIpath\fP. Defaults to stdout if \fIpath\fP is not given.
 
+.TP
+\fB--coverage-test\fP [\fIpath\fP]
+Argument is optional. Generate random attitudes for performance benchmark. Prints to \fIpath\fP. Defaults to stdout if \fIpath\fP is not given.
+
 .TP
```

Comment:
Add documentation for `time-based-seed` option. Make sure to mention that it overrides the `--generate-seed` option.